ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
-<!ENTITY      tab.signonsstored.label         "Сохранённые пароли">
-<!ENTITY      tab.signonsnotstored.label      "Пароли никогда не будут сохраняться">
-<!ENTITY      tab.nopreview.label             "Предварительно введённые в форму данные никогда не будут сохраняться">
-<!ENTITY      tab.nocapture.label             "Введённые в форму данные никогда не будут сохраняться">
+<!ENTITY      tab.signonsstored.label         "Сохраненные пароли">
```

Comment:
@yshchebet  @Kostrikov

Well sadly, we have to work with what we have.  

But consistency is important.  ll

@yshchebet Determine what is best, and make sure it implemented everywhere. Perhaps you can do a global find in Atom.